vm/vm.go

```diff
@@ -168,6 +187,60 @@ type CallInfo struct {
 	Calldata        []felt.Felt
 }
 
+type FeeTokenAddresses struct {
+	EthFeeTokenAddress  felt.Felt
+	StrkFeeTokenAddress felt.Felt
+}
```

Comment:
A bit opinionated here, but I think we should:
1. Either use the `starknet.FeeTokenAddresses` directly or write a small adapter to convert between 2 types. 
2. Move `FeeTokenAddressesFromGateway` the calculation to another place, because `vm` shouldn't call feeder gateway.